roles/dhcp/templates/dhcp_conf.j2

```diff
@@ -24,6 +24,21 @@ host {{ entry.host }}  {
 
 {% endfor %}
 
+
```

Comment:
@day4skiing this is fixing the main issue with booting from a secondary subnet ... as the class definition is global, it shouldn't be included in every subnet declaration as that will provide the wrong default gw to the consecutive subnet declarations after the first one